README.md

```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/JiejayLan/unifreefrontend.svg?branch=develop)](https://travis-ci.org/JiejayLan/unifreefrontend)
 
-[![Coverage Status](https://coveralls.io/repos/github/JiejayLan/unifreefrontend/badge.svg?branch=develop)](https://coveralls.io/github/JiejayLan/unifreefrontend?branch=develop)
+[![Coverage Status](https://coveralls.io/repos/github/JiejayLan/unifreefrontend/badge.svg?branch=test-jie)](https://coveralls.io/github/JiejayLan/unifreefrontend?branch=test-jie)
```

Comment:
I think the default is develop branch, so the icon should show the coverage of develop branch